repos/fdbt-site/tests/pages/ticketConfirmation.test.tsx

```diff
@@ -263,8 +257,7 @@ describe('pages', () => {
                         },
                     },
                 });
-                const numberOfElementsDueToProducts = mockSingleProduct.length;
-                const totalExpectedLength = 3 + numberOfElementsDueToProducts;
+                const totalExpectedLength = 6;
```

Comment:
Could we make this consistent with the previous test and just pass 6 directly in?

Reply:
Done 👍